scripts/delegateCommand.js

```diff
@@ -0,0 +1,20 @@
+const { spawn } = require('child_process');
```

Comment:
The modularization PR introduced the same approach with the ["scripts" folder](https://github.com/handsontable/handsontable/tree/feature/issue-7402/scripts). But, there script names uses hyphen as delimiter. Can we rename all the scripts within the PR to the same format?